packages/probing-frontend/src/router/routes.tsx

```diff
@@ -18,7 +18,7 @@ export const { routes, reactRouterDOMRoutes, hooks, components, utils } = new In
 })
   .addRoute({
     key: 'HOME',
```

Comment:
Let's modify the route key to a more descriptive name, like... `MONITORING_E_SERVICE_LIST`, or idk, any ideas?

Reply:
MONITORING_E_SERVICE_LIST seems legit